.github/workflows/debos.yml

```diff
@@ -95,4 +95,35 @@ jobs:
           # instruct fileserver to publish this directory
           url="${FILESERVER_URL}/${id}/"
           curl -X POST -H 'Accept: text/event-stream' "${url}"
+      - name: Prepare LAVA test job
+        run: |
+          id="${GITHUB_REPOSITORY}-${GITHUB_RUN_ID}-${GITHUB_RUN_ATTEMPT}"
+          url="${FILESERVER_URL}/${id}/"
+          BUILD_FILE_NAME=flash.tar.gz
+          BUILD_DOWNLOAD_URL="${url}${BUILD_FILE_NAME}"
+          LAVA_JOB_FILE="../ci/lava/rb3gen2-vision-kit-boot.yaml"
+
+          sed -i "s|{{BUILD_FILE_NAME}}|${BUILD_FILE_NAME}|g" ${LAVA_JOB_FILE}
```

Comment:
I find it weird to in-place sed source files, I would make a copy first, but I understand this is copy-pasted from another instance

I'd also quote `"${LAVA_JOB_FILE}"`